feconf.py

```diff
@@ -839,3 +839,43 @@ def get_empty_ratings():
 VIEW_METHOD_USERNAME = 'username'
 
 QUESTION_BATCH_SIZE = 10
+
+# Valid parent-child relation in TextAngular.
+ALLOWED_PARENT_LIST = {
```

Comment:
You might want to do it like this instead:

```

RTE_CONTENT_SPECS = {
    RTE_TYPE_TEXTANGULAR: {
        ALLOWED_PARENT_LIST = {...},
        ALLOWED_TAG_LIST = {...},
    },
    RTE_TYPE_CKEDITOR = {
        ...
    }
}
```

where RTE_TYPE_TEXTANGULAR etc are constants. Then you can just swap out RTE_TYPE_TEXTANGULAR for RTE_TYPE_CKEDITOR later and everything else should just work.

Reply:
Done!